### ISSUE TITLE
Multi-class 3D segmentation visualization in Patch-Based training

### ISSUE THREAD

**Issue by Nastii22** [fepegar/torchio-notebooks#11]
Hello @fepegar ,

I'm working on a multi-class 3D segmentation project which contains 3 classes, including background. 
I've utilized Medical image segmentation with TorchIO, MONAI & PyTorch Lightning the example provided in your GitHub repository (https://github.com/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_MONAI_PyTorch_Lightning.ipynb). The key difference in my approach is that I'm training the model in a patch-based manner as provided in your GitHub repository (https://github.com/fepegar/torchio-notebooks/blob/main/notebooks/TorchIO_tutorial.ipynb).

Could you help me understand how to modify the testing part of the patch-based method to fit multi-class segmentation? Specifically, I need to visualize and check the model output for each class alongside the input image and the corresponding labels for each class.

Your assistance would be greatly appreciated.

**Comment by fepegar** [comment]
Closing in favor of https://github.com/fepegar/torchio/discussions/1177
